pkgs/development/compilers/terra/default.nix

```diff
@@ -1,68 +1,87 @@
-{ lib, stdenv, fetchurl, fetchFromGitHub
-, llvmPackages, ncurses, lua
-}:
+{ lib, stdenv, fetchurl, fetchFromGitHub, llvmPackages, ncurses, cmake, libxml2
+, symlinkJoin, breakpointHook, cudaPackages, enableCUDA ? false }:
 
 let
-  luajitArchive = "LuaJIT-2.0.5.tar.gz";
+  luajitRev = "9143e86498436892cb4316550be4d45b68a61224";
+  luajitArchive = "LuaJIT-${luajitRev}.tar.gz";
   luajitSrc = fetchurl {
-    url    = "http://luajit.org/download/${luajitArchive}";
-    sha256 = "0yg9q4q6v028bgh85317ykc9whgxgysp76qzaqgq55y6jy11yjw7";
+    url = "https://github.com/LuaJIT/LuaJIT/archive/${luajitRev}.tar.gz";
+    sha256 = "0kasmyk40ic4b9dwd4wixm0qk10l88ardrfimwmq36yc5dhnizmy";
```

Comment:
```suggestion
    owner = "LuaJIT";
    repo = "LuaJIT";
    rev = luajitRev;
    sha256 = "1zw1yr0375d6jr5x20zvkvk76hkaqamjynbswpl604w6r6id070b";
```

Reply:
Got this fixed in [`c49d9ad` (#125896)](https://github.com/NixOS/nixpkgs/pull/125896/commits/c49d9adc385ccff57ef020456f3700fe6878619d).